src/routes/(app)/+page.server.ts

```diff
@@ -40,3 +41,21 @@ export const load = async ({ locals }) => {
 		layoutReset: cmsPage.fullScreen
 	};
 };
+
+export const actions = {
+	navigate: async ({ locals }) => {
+		await collections.sessions.findOneAndUpdate(
+			{
+				sessionId: locals.sessionId
+			},
+			{
+				$set: {
+					updatedAt: new Date(),
+					acceptAgeLimitation: true
```

Comment:
please set the `expiresAt` field - like in the originall code from `hooks.server.ts`.